setup/github_codespaces.md

```diff
@@ -0,0 +1,20 @@
+# Github codespaces
+A [codespace](https://docs.github.com/en/codespaces/overview) is a development environment that's hosted in the cloud.
+
+# Creating your codespace
+* Fork the [IRIS-RoR-Bootcamp-2022](https://github.com/IRIS-NITK/IRIS-RoR-Bootcamp-2022) repo.
+
+<img width="695" alt="Screenshot 2022-12-21 212710" src="https://user-images.githubusercontent.com/66632353/208948037-8bbf1260-e35d-462b-b5af-15040458e9d3.png">
+
+* Click on **Code** and from the codespaces dropdown click on the ➕ icon to create a new codespace.

```

Comment:
Again, keep the text describing the action first and the image. Here it looks like the image is telling them how to fork.